src/served/status.cpp

```diff
@@ -20,4 +20,4 @@
  * SOFTWARE.
  */
 
-#include <served/status.hpp>
\ No newline at end of file
+#include <served/status.hpp>
```

Comment:
Was anything changed? I don't think this line should be committed

Reply:
It's better to have a newline at end of file